wazo_confgend/templates/asterisk/extensions/ivr.jinja

```diff
@@ -13,7 +13,8 @@ same  =   n,Goto(t,1)
 {%- endif %}
 
 {% for choice in ivr.choices %}
-exten = {{ choice.exten }},1,Gosub(forward,s,1({{ choice.destination.gosub_args | replace(";", "\;") }}))
+exten = {{ choice.exten }},1,CELGenUserEvent(WAZO_IVR_CHOICE, {"xivo-ivr-{{ ivr.id }}": "{{ choice.exten }}"})
+exten = {{ choice.exten }},2,Gosub(forward,s,1({{ choice.destination.gosub_args | replace(";", "\;") }}))
```

Comment:
Use `n`, it will increment automatically
```suggestion
exten = {{ choice.exten }},n,Gosub(forward,s,1({{ choice.destination.gosub_args | replace(";", "\;") }}))
```

Reply:
Sure will use n instead.